server/command/service.go

```diff
@@ -53,14 +53,22 @@ func (s *service) ExecuteCommand(pluginContext *plugin.Context, commandArgs *mod
 	if pluginContext == nil || commandArgs == nil {
 		return errorOut(params, errors.New("invalid arguments to command.Handler. Please contact your system administrator"))
 	}
+
+	enableOAuthServiceProvider := s.api.Mattermost.Configuration.GetConfig().ServiceSettings.EnableOAuthServiceProvider
+	if enableOAuthServiceProvider == nil || !*enableOAuthServiceProvider {
+		return errorOut(params, errors.Errorf("the system setting `Enable OAuth 2.0 Service Provider` need to be enabled in oder for the Apps plugin to work. Please go to %s/admin_console/integrations/integration_management and enable it.", commandArgs.SiteURL))
```

Comment:
1/5 to use the "com.mattermost.apps" name instead of (or in addition to)  "Apps plugin" above.